### PR TITLE
Fixed indent function

### PR DESCRIPTION
**Pull request by Bchass**
Updated `indent` function, it's working correctly outside the project when tested against a `tree` and inside the project. Drafting the PR until we can confirm it's working correctly with someone else.

### CODE REVIEW

File: squirrel/xml.py [hunk 1]
```diff
@@ -48,7 +48,41 @@ def indent(tree, space=" ", level=0):
...
+            if len(child):
+                indent_child(child, child_level)
+            if not child.tail or not child.tail.strip():
+                child.tail = child_indentation
+
+        # Dedent and then overwite preivous indent
+        if not child.tail.strip():
+            child.tail = indentations[level]
+
+    indent_child(tree, 0)
```
Comment (zeddo123) [inline squirrel/xml.py:84]: Okay! This function works for me.

**Review by zeddo123 — APPROVED** [review]

**Comment by zeddo123** [comment]
I've tested your branch on my machine, and it's working correctly. Though you still have the old function in your branch.

https://github.com/Bchass/squirrel/blob/05a1634b9ca0f4d965c6019cd88829387a8499e8/squirrel/xml.py#L37-L49

Git wise, there's a merge commit that we should remove before merging. otherwise, LGTM thank you!

**Comment by Bchass** [comment]
Between Friday and today my git is being pretty funky and is throwing fatal errors at me. I'm going to have to re-install git and then redo the commit/PR. I promise this will be the last one I have to redo :sweat_smile:

**Comment by zeddo123** [comment]
> Between Friday and today my git is being pretty funky and is throwing fatal errors at me. I'm going to have to re-install git and then redo the commit/PR. I promise this will be the last one I have to redo sweat_smile

No problem! I can related to struggling with git. Take your time, and thank you for contributing.

**Pull request closed without merging**